client/my-sites/plugins/no-permissions-error.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import EmptyContent from 'components/empty-content';
+import DocumentHead from 'components/data/document-head';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+
+function NoPermissionsError( { translate } ) {
```

Comment:
@gwwar: I'm fine with the component name (because I can't figure out a better one), but perhaps you might have tips for a better one?